.github/actions/waffles/waffles.py

```diff
@@ -112,6 +118,12 @@ def _get_flask_endpoints(flask_app: Flask) -> List[URL]:
     for rule in flask_app.url_map.iter_rules():
         endpoint = URL(rule.rule)
         endpoints.append(endpoint)
+
+    for extra_endpoint in flask_app.config.get("EXTRA_ROUTES", []):
+        endpoint = URL(extra_endpoint)
+        endpoints.append(endpoint)
+
+    return endpoints
```

Comment:
It seems we have an extra return statement.

Reply:
whoops! good catch 👍